### PR TITLE
fix menu

### PR DESCRIPTION
**Pull request by duythien**

### CODE REVIEW

File: apps/controllers/ControllerAdminBase.php [hunk 1]
```diff
@@ -136,6 +144,73 @@ public function initialize()
...
+                                                "code" => "logo",
+                                                "name" => "Change Logo",
+                                                "sub"  => []
+                                            ]
+                                        ]
+                                    );
+        $this->menuStruct[] = array("code" => "update",
+                                    "name" => "Updates",
+                                    "sub" => array()
+                                    );
```
Comment (duythien) [inline apps/controllers/ControllerAdminBase.php]: Could you move it to a function

File: apps/controllers/ControllerAdminBase.php [hunk 2]
```diff
@@ -136,6 +144,73 @@ public function initialize()
         $this->view->currentOrder = $this->currentOrder;
         $this->view->setTemplateBefore('admin');
         $this->loadDefaultAssets();
+
+        /*============================================
+        =            Set menu struct data            =
```
Comment (duythien) [inline apps/controllers/ControllerAdminBase.php]: @vmtu This comment should be into Dock block below

File: apps/controllers/ControllerAdminBase.php (hunk 1 above)
Comment (glmanhtu) [inline apps/controllers/ControllerAdminBase.php]: I think this code will not reusing in the future.  Therefore, we no need using function any more

Comment (sergeyklay) [inline apps/controllers/ControllerAdminBase.php]: If so, we can store this structure into config or db table (e.g. `menu`) and use somethig like this:

``` php
$di->setShared('menuStruct', function() use ($config) {
    // if structure received from db table instead getting from $config
    // we need to store it to cache for reducing db connections
    $struct = $config->get('menuStruct')->toArray();

    return $struct;
});

// ...

public function initialize()
{
    $this->view->menuStruct = $this->menuStruct;
}
```

Comment (duythien) [inline apps/controllers/ControllerAdminBase.php]: :+1:

**Pull request merged**